Classes/Issues/IssueManagingContextController.swift

```diff
@@ -91,6 +93,16 @@ final class IssueManagingContextController: NSObject, ContextMenuDelegate {
             if result.pullRequest {
                 actions.append(.reviewers)
             }
+            
+            let viewerCanUpdate =
+                editIssueTitleViewControllerDelegate?
+                .viewerCanUpdate
+                ?? false
+            
+            if viewerCanUpdate {
```

Comment:
Since the result of this `Bool` isn't used again, can it just all be in the `if` statement? No need for a variable